src/iceberg/CMakeLists.txt

```diff
@@ -137,13 +137,23 @@ if(ICEBERG_BUILD_BUNDLE)
        "$<IF:$<TARGET_EXISTS:Parquet::parquet_static>,Parquet::parquet_static,Parquet::parquet_shared>"
        "$<IF:$<TARGET_EXISTS:avro-cpp::avrocpp_static>,avro-cpp::avrocpp_static,avro-cpp::avrocpp_shared>"
   )
+  if(ICEBERG_BUILD_REST_CATALOG)
+    list(APPEND ICEBERG_BUNDLE_STATIC_BUILD_INTERFACE_LIBS
+         "$<IF:$<TARGET_EXISTS:CURL::libcurl>,CURL::libcurl,CURL::libcurl>"
+         "$<IF:$<TARGET_EXISTS:cpr::cpr>,cpr::cpr,cpr::cpr>")
+  endif()
```

Comment:
```suggestion
```

`libiceberg-bundle` does not interfere with rest catalog so it should not depend on curl or cpr.

Reply:
Same for below.